Ryujinx.Audio/Renderer/Dsp/Command/UpsampleCommand.cs

```diff
@@ -40,6 +40,8 @@ public UpsampleCommand(uint bufferOffset, UpsamplerState info, uint inputCount,
                 info.InputBufferIndices[i] = (ushort)(bufferOffset + inputBufferOffset[i]);
             }
 
+            Array.Resize(ref info.BufferStates, Math.Max(info.BufferStates.Length, (int)inputCount));
+
```

Comment:
We already initialize array for ``InputBufferIndices`` and others, so for consistency:

```suggestion
            info.BufferStates = new UpsamplerBufferState[Math.Max(info.BufferStates.Length, (int)inputCount)];
```

Reply:
Was originally intending to keep previous state.